csrc/pf_types.h

```diff
@@ -23,6 +23,13 @@
 ** Type Declarations
 ***************************************************************/
 
+#ifndef AMIGA
+#include <sys/types.h>
+#endif
+
+/* file_offset_t is used in place of off_t */
+typedef long file_offset_t;
```

Comment:
Based on what I am seeing in the gForth code base, I think it might make sense to use `size_t` instead. I am in the middle of testing both `long` and `size_t` on Windows.

Reply:
I guess that isn't going to work, this type needs to be signed.